test/upgrade/run-upgrade.sh

```diff
@@ -14,22 +14,37 @@ LATESTVERSION=$(./node_modules/.bin/ts-node src/scripts/latestVersion.ts)
 AVAILABLEVERSIONS=$(./node_modules/.bin/ts-node src/scripts/availableVersions.ts)
 CURRENTGITSHA=$(git rev-parse HEAD)
 
-# Connection string
-CONNSTR="postgres://postgres:test@127.0.0.1:5432/to_upgrade"
-
 # Github Actions apparently doesn't pull down the tags by default?
 git pull origin --tags ${CURRENTGITSHA}
 
 IFS=',' read -r -a availableVersions <<<"$AVAILABLEVERSIONS"
 len=${#availableVersions[@]}
 i=0
 while [ $i -lt $(($len - 1)) ]; do
+  # Clear mutations, if any
+  git reset --hard HEAD
   echo "Upgrading from version ${availableVersions[$i]} to ${LATESTVERSION}"
   # Check out the older version of the codebase and launch the engine with a local postgres
   git checkout v${availableVersions[$i]}
-  yarn docker-compose &
+  mkdir -p ${PWD}/../db
+  if [ -f "${PWD}/docker-compose.yml" ]; then
```

Comment:
should this be changed with the CONTRIBUTING.md?

Reply:
No, here we are figuring out if the repo has been switched back to an earlier version that still uses Docker compose or not.